tests/components/home_connect/test_sensor.py

```diff
@@ -103,11 +103,11 @@ async def test_sensors(
 # Entity mapping to expected state at each program sequence.
 ENTITY_ID_STATES = {
     "sensor.dishwasher_operation_state": (
-        "Delayed",
-        "Run",
-        "Run",
-        "Run",
-        "Ready",
+        "delayed_start",
+        "run",
+        "run",
+        "run",
+        "ready",
```

Comment:
Do these changes now also change the possible states?

Reply:
Yep, because translation keys cannot be in camel caps...

Reply:
That's true. I didn't realize this was a breaking change.